code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -254,7 +254,7 @@
 /obj/item/weapon/gun/smartgun/able_to_fire(mob/living/user)
 	. = ..()
 	if(.)
-		if(!ishuman(user)) 
+		if(!ishuman(user))
```

Comment:
yeah avoid committing whitespace changes like this in future.

Reply:
My bad, I didn't know how to commit only one line, I found how to do it now though so its good now.

Reply:
its fine for this just as a note for future to avoid it where possible.